code/_core/turf/unsimulated/generation/jungle_deep.dm

```diff
@@ -52,16 +52,38 @@
 			if(prob(5))
 				new /obj/marker/generation/jungle_rock_grass(src)
 			else if(prob(1))
+				new /obj/marker/generation/jungle_tree_small(src)
+			else if(prob(1))
+				new /obj/marker/generation/jungle_high_grass/small(src)
+			if(prob(1))
 				new /obj/marker/generation/jungle_light_stick(src)
 			else if(prob(1))
+				new /obj/marker/generation/mob/arachnid(src)
+			else if(prob(1))
+				new /obj/marker/generation/jungle_high_grass/small(src)
+			if(prob(1))
```

Comment:
In some instances you're breaking the else if chain. You should be careful with this and make sure you mean else if and not just if as it may cause objects to spawn inside eachother.

Reply:
For whatever reason when they *were* else if the mobs wouldnt spawn at all, OR trees/plants in the area wouldn't spawn.

Reply:
![image](https://user-images.githubusercontent.com/61367602/125584174-fbbbd929-9002-4ecd-aabc-be2da21e410b.png)
Im not saying items *dont* spawn in eachother, but you seriously need to go far out of your way to find them